client/my-sites/plugins/plugin-management-v2/update-plugin/index.tsx

```diff
@@ -58,23 +60,42 @@ export default function UpdatePlugin( {
 
 	let content;
 
+	const allStatuses = getPluginActionStatuses( state );
+
+	const updateStatuses = allStatuses.filter(
+		( status ) => status.pluginId === plugin.id && status.action === UPDATE_PLUGIN
+	);
+
 	if ( ! allowedActions?.autoupdate ) {
 		content = <div>{ translate( 'Auto-managed on this site' ) }</div>;
+	} else if ( updateStatuses.length > 0 ) {
+		content = (
+			<div className="update-plugin__plugin-action-status">
+				<PluginActionStatus
+					showMultipleStatuses={ false }
+					currentSiteStatuses={ updateStatuses }
+					selectedSite={ selectedSite }
+				/>
+			</div>
+		);
 	} else if ( hasUpdate ) {
 		content = (
-			<>
-				<Badge className="update-plugin__badge" type="warning">
-					{ updated_versions[ 0 ] } { translate( 'Available' ) }
-				</Badge>
-				<Button
-					className="update-plugin__plugin-update-button"
-					borderless
-					compact
-					href={ `/plugins/${ plugin.slug }` }
-				>
-					{ translate( 'Update' ) }
-				</Button>
-			</>
+			<div className="update-plugin__plugin-update-wrapper">
+				<span className="update-plugin__current-version">{ plugin?.version }</span>
```

Comment:
For some reason, `plugin.version` sometimes does not hold the current version of the plugin, but the target one.

<img width="1313" alt="image" src="https://user-images.githubusercontent.com/3418513/188774308-aa31a3c8-2a7c-43aa-b4e1-37b537fc009c.png">